### ISSUE TITLE
Running distro failed with "Installation directory not found" error if executable and distro names are mismatched case-sensitive

### ISSUE THREAD

**Issue by evakili** [yuk7/wsldl#39]
I have a Debian WSL distro on my old computer and move it to my new PC with the following process:

```bat
:: On the source computer
wsl.exe --export debain Debian-Export.tar
```
```bat
:: On the destination computer
wsl.exe --import debian D:\some\path Debian-Export.tar
```

Everything is OK but the official Debian launcher app is not installed on destination. So I want to use this launcher app instead.

Launcher's `run` and `config` commands are run successfully. But launching the distro with renamed `launcher.exe` failed with the following error:

```
C:>debian.exe
Installation directory not found: .
Make sure it exists or reinstall.
```

**Comment by yuk7** [comment]
I perfomed the some steps but could not confirm the same problem...
on wsldl 19031901,Win10 1903

What versions are you using?
Please note that the excuteble file name and instance name is case sensitive.

**Comment by evakili** [comment]
I found the problem: the code compares registry value for distro name with executable name as case-sensitive. My distro name is `Debian` and executable is `debian.exe`.
Look at this [line](https://github.com/yuk7/wsldl/blob/2d4e33691db4e047db924fcd6d63070413a9aa45/wsld.h#L131):
```cpp
if((subKeySz == UUID_SIZE) && (wcscmp(regDistName,DistributionName)==0))
```
Change it as below solves the problem:
```cpp
if((subKeySz == UUID_SIZE) && (_wcsicmp(regDistName,DistributionName)==0))
```

**Comment by evakili** [comment]
Oh! I post the comment before reading your comment! Is there any reason for case-sensitivity?

**Comment by yuk7** [comment]
The official launcher hardcodes the name.
wsldl isn't so.

**Comment by yuk7** [comment]
Ok,I'll fix it.

**Comment by yuk7** [comment]
fixed on f74ad77.
Thank you!🙇

If you need the latest test build, you can download it here:
https://ci.appveyor.com/project/yuk7/wsldl
